js/ui/components/clubhouse.js

```diff
@@ -378,12 +380,23 @@ var ClubhouseComponent = new Lang.Class({
             Main.layoutManager.addChrome(this._banner.actor);
             this._banner.reposition();
         }
+
+        // Sync the visibility here because the screen may be locked when a notification
+        // happens
+        this._syncVisibility();
```

Comment:
I'm not sure I understand why this is here; if the screen is locked when a notification happens, we will have been disabled before. Upon unlocking, we will be re-enabled and the banner will become visible, if it was triggered. Am I missing something?